### DISCUSSION TITLE
InboxSDK and Vite

### DISCUSSION THREAD

**Discussion by pmamatsis** [InboxSDK/InboxSDK#1138] [Q&A]
Hello everyone,
   I am trying to create a simple chrome extension using Vue 3 and Vite. I am seeing the _Hello World_ example is using webpack. Can I use InboxSDK with Vite? I know that my question is a bit abstract but I am trying to learn how to create Chrome extensions. Thank you so much in advance.

**Comment by wegry** [comment]
Hey @pmamatsis, I wouldn't expect vite to have issues with including the SDK. Multiple entry points, that is one for `@inboxsdk/pageWorld.js` and `@inboxsdk/background.js`, seem like they'd be the only tricky bit.

If you give using vite a try, we'd be open to PRs over at [InboxSDK/hello-world](https://github.com/InboxSDK/hello-world) with a vite example. If you run into issues in the process, feel free to file a bug.
